lms/templates/instructor/instructor_dashboard_2/student_admin.html

```diff
@@ -147,6 +147,9 @@ <h5 class="hd hd-5">${_("Task Status")}</h5>
 </div>
 %endif
 
+%endif
+
+%if section_data['access']['instructor']:
```

Comment:
The Rescore for ALL Learner's for Entrance Exam should also be available to only instructors.  That happens above: https://github.com/edx/edx-platform/blob/14081a1bb9387e7201a085ab53a7a90eca942e38/lms/templates/instructor/instructor_dashboard_2/student_admin.html#L116